### ISSUE TITLE
Bootstrap Templates :: Class `form-control` beibehalten

### ISSUE THREAD

**Issue by tbaddade** [yakamara/redaxo_yform#161]
Gibt man eine eigene Class via `{'class': 'my-class'}` an, geht `form-control` verloren.

**Comment by dergel** [comment]
ja.. die werden ersetzt und müssen deshalb dann selbst nochmal gesetzt werden. das ist ok so finde ich. sonst ist dieses attribute feld ziemlich umständlich zu betrachten - weil man im prinzip alles überschreiben kann (und deswegen keine einzelnen ausnahmen sein sollten)

**Comment by alexplusde** [comment]
welche Felder betrifft das? Könnte man in der Doku darauf hinweisen.

**Comment by tbaddade** [comment]
Ich finde, wenn man das Bootstrap-Template verwendet sollte auch Bootstrap herauskommen. Alles andere kann überschrieben werden, jedoch nicht die Klasse `form-control`. Die sollte zwingend immer gesetzt sein.

**Comment by alexplusde** [comment]
+1, und wenn man es wenigstens in das notice-Feld zum direkten reinkopieren schreibt und nicht immer wieder nachschlagen muss.

**Comment by dergel** [comment]
hmm.. vielleicht ist es doch ok, wenn das im bootstrap template dann immer gesetzt wird. gerne als PR, oder @tbaddade direkt reinbauen.

**Comment by tbaddade** [comment]
Würde ich so erst einmal nicht angehen. Wegen BS4 müsste man vermutlich sowieso die ytemplates umbauen.

**Comment by alexplusde** [comment]
Ist ein Wechsel auf Bootstrap 4 denn geplant? Sind die schon aus der Alpha draußen?

**Comment by gharlan** [comment]
Nicht nur aus alpha, sondern auch aus der Beta sind sie raus seit 5 Tagen.

**Comment by dergel** [comment]
jetzige ist bs3 und daher sollte das dort auch entsprechend rein .. bs4 templates müssten wir dann komplett neu aufbauen

**Comment by tbaddade** [comment]
> jetzige ist bs3 und daher sollte das dort auch entsprechend rein

Es gibt ja eine Möglichkeit, die Class zu notieren. Die BS3 Templates würde ich dahingehend nicht mehr anpassen. Wenn jemand dennoch einen PR bereitstellt, dann sehr gern.

**Comment by dergel** [comment]
gerne als PR.. ist dann im Prinzip ausschliesslich in den bs3 templates anzupassen.
